From 8697948f52f2f6f7af25e4c7cb2ecc0f28967da8 Mon Sep 17 00:00:00 2001
From: shalini-mitt <shalini.mittal20@st.niituniversity.in>
Date: Sat, 14 Jan 2023 23:15:37 +0530
Subject: [PATCH 01/15] first commit

---
 README.md | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..c4567f3
--- /dev/null
+++ b/README.md
@@ -0,0 +1 @@
+"# first" 
-- 
2.39.2.windows.1

